### ISSUE TITLE
[ANNOTATION] Please add metrics class wise 

### ISSUE THREAD

**Issue by sumanttyagi** [LADI-Dataset/ladi-overview#11]
can you share the metrics of each class ?
and also when you say the damage building is it from the fire , earthquake or hurricanes FEMA has all these does you sample of 10k was including all of these ?
Great work !!

**Comment by jeffliu-LL** [comment]
Thanks for your interest. More complete dataset statistics will be available in a few weeks when the writeup is finished.

In terms of hazard types represented, the dataset includes imagery from the following event types: hurricanes, floods, severe storms, fires, and tornadoes. Hurricanes are the most well-represented event type due to the nature of the data. Performance on hurricane-like damage, such as from severe storms and tornadoes, seems to generalize well for those event types. However, fires are not very well represented, so I would caution using this dataset/classifier as-is if you are pursuing a fire-related application.

**Comment by jeffliu-LL** [comment]
The class metrics and other information are available in our writeup, available on arxiv: https://arxiv.org/abs/2406.02780

Please let us know if you have any additional questions!

**Comment by jeffliu-LL** [comment]
There's a small number of examples of tree_damage directly from tornado events. If you look at figure 3b of the paper https://arxiv.org/pdf/2406.02780, about 0.3% of the images in the training/validation set have tree_damage, which is about 27 examples. There's also a larger number of tree damage examples associated with hurricanes, about 1700, in the train/val set, which may partially overlap with the kind of damage you'd expect from tornadoes.
